src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
So I assume it was using the Internal one before your change? If so I'm good with this as a temporary workaround but please file an issue tracking the right fix for this project. I would like either @davidsh or @CIPop to suggest the best long term solution for fixing the issue.